packages/next-codemod/bin/__testfixtures__/peer-dep-out-of-range/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "peer-dep-out-of-range",
+  "scripts": {
+    "dev": "next dev --turbopack"
+  },
+  "dependencies": {
+    "next": "15.0.0-canary.0",
+    "react": "19.0.0-rc.0",
+    "react-dom": "19.0.0-rc.0",
+    "unmet-prerelease": "file:./unmet-prerelease",
+    "unmet-range": "file:./unmet-range"
```

Comment:
Smart way to test this without having to hunt for matching packages on NPM.

Let's also add a package where it does match e.g. one with `"react": "^19.0.0-rc.0"`.